manim/mobject/geometry.py

```diff
@@ -812,3 +859,233 @@ class RoundedRectangle(Rectangle):
     def __init__(self, **kwargs):
         Rectangle.__init__(self, **kwargs)
         self.round_corners(self.corner_radius)
+
+
+class Tiling(VMobject):
+    """
+    The purpose of this class is to create tilings/tesselations.
+    Tilings can also be seemlessly transformed into each other.
+    This requires their ranges to be the same, tiles to be oriented and
+    rotated the correct way, as well as having a vertex setup that
+    allows proper transformations.
+    
+    Parameters
+    ----------
+    tile_prototype : Mobject or function(x,y) that returns a Mobject
```

Comment:
I suppose x, y are ints?
```suggestion
    tile_prototype : Union[:class:`~.Mobject`, Callable[[:class:`int`, :class:`int`], :class:`~.Mobject`]]
```
also, maybe you should add a few short descriptions to each one